src/go/k8s/tests/README.md

```diff
@@ -23,6 +23,9 @@ If set, do not delete the mocked control plane or kind cluster.
 --skip-delete
 If set, do not delete resources created during tests (helpful for debugging test failures, implies --skip-cluster-delete).
 To run a single e2e test and keep the kind cluster and the kutt
+
+--test=<test-directory-to-run> 
+If set, will run tets found in <test-directory-to-run>.
```

Comment:
If the `TEST_NAME` environment variable is set, then `--test` is used. 

https://github.com/redpanda-data/redpanda/blob/8a9d43faa4b0d998eb6cd79521090b1733eca0e1/src/go/k8s/Makefile#L26-L28

Reply:
Yep, its just there as a helpful command when running this locally at times.